techniques/shortchecklist.html

```diff
@@ -141,8 +141,8 @@ <h2 class="notoc">Useful links</h2>
 
 
 <section>
-  <p>This page points developers of specifications to various aspects of their spec that may need internationalization review. It can also be used by spec reviewers, to get an idea of what to look for in a spec.</p>
-<p><strong>You should raise an issue in your own repository to show the answers to these questions, and attach the i18n-tracker label so that we are notified. Under 'Create a Report' you'll find a way to create a report.</strong></p>
+  <p><strong>The primary purpose for this page is educational. It helps you to learn about internationalization features that may affect the design of your spec.  You should read it <em>and</em> <a href="https://www.w3.org/TR/international-specs/">Internationalization Best Practices for Spec Developers</a> as early as possible, so that you are aware of what your spec will need to address.  Then, to ensure that you haven't missed anything, you can also use this page as a checklist prior to publishing the spec as a FPWD.</strong></p>
+<p>You should raise an issue in your own repository to show the answers to these questions, and attach the i18n-tracker label so that we are notified. Under 'Create a Report' you'll find a way to create a report.</p>
```

Comment:
```suggestion
<p>You should raise an issue in your own repository to show the answers to these questions, and attach the <code>i18n-tracker</code> label so that we are notified. Under 'Create a Report' you'll find a way to create a report.</p>
```